AmazonChimeSDKDemo/AmazonChimeSDKDemo/MeetingModel.swift

```diff
@@ -638,7 +640,10 @@ extension MeetingModel: ActiveSpeakerObserver {
 
 extension MeetingModel: DataMessageObserver {
     func dataMessageDidReceived(dataMessage: DataMessage) {
-        chatModel.addDataMessage(dataMessage: dataMessage)
+        logger.info(msg: "Received data message for '\(dataMessage.topic)', \(dataMessage.text() ?? "")")
```

Comment:
why do we need extra logging?

Reply:
This is a place holder until we have the actual UI implementation.

Reply:
Could you add some comments/TODO here? Or update this bc UI is done.

Reply:
Good catch. Will remove.